### DISCUSSION TITLE
在馬斯克的Neuralink之前，腦機接口早在先前就已經試用在ALS患者上

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1050] [General]
那天看到Neuralink所發布的消息：「六個月內展開腦機介面植入人腦試驗」
覺得這項科技能為那些禁錮在軀殼裡的靈魂帶來一絲希望，
是非常令振奮的突破！
https://udn.com/news/story/6813/6806951

查詢資料時發現其實在馬斯克之前就已經有相關試驗，
也已經應用於人體上，意外呼應了上方報導中最後一段的科學家抨擊(?
好好笑
https://inewsdb.com/%E6%B1%BD%E8%BB%8A/%E9%80%99%E5%80%8B%E8%85%A6%E6%A9%9F%E6%8E%A5%E5%8F%A3%E5%B7%B2%E7%B6%93%E8%A9%A6%E7%94%A8%E5%9C%A8als%E6%82%A3%E8%80%85%E4%B8%8A%EF%BC%8C%E6%AF%94%E9%A6%AC%E6%96%AF%E5%85%8B%E7%9A%84neuralink/

總之還是希望這項科技真正趕快成熟，
讓難以順利表達想法的病患們能夠透過相關科技掙脫疾病與身體軀殼的束縛！

**Comment by BlahFish1006** [comment]
馬斯克真的是個一直敢於行走在風口浪尖最前端的男人，將最新穎、不可思議的想法帶到大眾面前討論。
但蠻意外Neurolink竟不是第一個將BCI技術實際應用於人體試驗上並獲得FDA認證的機構，
下方的連結介紹得非常清楚，該家公司的腦機支架Stentrode，甚至不需要切開人的頭骨或破壞組織，而是透過血管植入，真的太厲害了！
希望這項技術能早日被實際應用於臨床上，幫助患者脫離束縛。
